install.sh

```diff
@@ -2,12 +2,16 @@ cd ~
 git clone https://github.com/YetAnotherPackageInstaller/rewrite.git YAPI --depth 1
 git clone https://github.com/pybee/toga.git
 cd toga
-pip install -e src/core
-pip install -e src/dummy
-pip install -e src/gtk
+pip3 install -e src/core
+pip3 install -e src/dummy
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+  pip3 install -e src/gtk
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+  pip3 install -e src/cocoa
+fi
```

Comment:
I dont know why but gui doesnt work. This is the error:
```Obtaining file:///home/wabri/toga/src/core
Requirement already satisfied: travertino>=0.1.0 in /home/wabri/git/rewrite/environment/lib/python3.5/site-packages (from toga-core==0.3.0.dev12) (0.1.2)
Installing collected packages: toga-core
  Found existing installation: toga-core 0.3.0.dev12
    Uninstalling toga-core-0.3.0.dev12:
      Successfully uninstalled toga-core-0.3.0.dev12
  Running setup.py develop for toga-core
Successfully installed toga-core
Obtaining file:///home/wabri/toga/src/dummy
Requirement already satisfied: toga-core==0.3.0.dev12 in ./src/core (from toga-dummy==0.3.0.dev12) (0.3.0.dev12)
Requirement already satisfied: travertino>=0.1.0 in /home/wabri/git/rewrite/environment/lib/python3.5/site-packages (from toga-core==0.3.0.dev12->toga-dummy==0.3.0.dev12) (0.1.2)
Installing collected packages: toga-dummy
  Found existing installation: toga-dummy 0.3.0.dev12
    Uninstalling toga-dummy-0.3.0.dev12:
      Successfully uninstalled toga-dummy-0.3.0.dev12
  Running setup.py develop for toga-dummy
Successfully installed toga-dummy
./install.sh: 7: ./install.sh: [[: not found
./install.sh: 9: ./install.sh: [[: not found
Collecting requirements.txt
  Could not find a version that satisfies the requirement requirements.txt (from versions: )
No matching distribution found for requirements.txt
rm: cannot remove '.git*': No such file or directory
rm: cannot remove '.travis.yml': No such file or directory
rm: cannot remove '.circleci': No such file or directory
rm: cannot remove 'install.sh': No such file or directory
rm: cannot remove 'LICENSE': No such file or directory
rm: cannot remove 'README.md': No such file or directory
```
Maybe a new version of the gui need to be created.
I try to install toga with virtual environment and after in local but give me the same output.

Reply:
I will look into this. You also need to change your script to look for os type because linux uses src/gtk and macos uses src/cocoa.